pulp_ansible/app/galaxy/v3/serializers.py

```diff
@@ -22,8 +20,6 @@ class CollectionSerializer(serializers.ModelSerializer):
     class Meta:
         fields = (
             "href",
-            "created_at",
```

Comment:
Looks like a breaking change in API response format.

Reply:
It is but these fields are not in use by any clients and not in the API spec itself. These got added almost by accident when we implemented this API call the first time.

Reply:
CLI counts on these fields: https://pulp.plan.io/issues/7647

it will be introduced here:
https://github.com/ansible/ansible/pull/71904/files#diff-fe8d3e13b2884db3e502001befe5ca81b0b203fe2cec9b31e2ee9b0f14a8e835R673-R677

Reply:
I'm going to add these back along with a migration to populate the data from the oldest collection version.

Reply:
These have been added back. Also I talked with jborean who agreed having the values reflect the created time of the oldest and newest CollectionVersion's for that collection is what would be usable for the CLI. That's what this implementation does.